src/components/Requirements.vue

```diff
@@ -372,6 +399,13 @@ export default Vue.extend({
       });
       return (id < 2 || id === currentDisplay);
     },
+    showReqTooltip() {
+      /* Revisit this when implementing tooltips appearing for first time
+      if (this.req_tooltip_seen === false) {
+        tour.start();
+        // add update req tooltip seen
+      } */
+    },
```

Comment:
Let's not comment out code but leave the function available. Either comment it out completely or most ideally remove the entire section